build.pl

```diff
@@ -81,11 +80,22 @@ sub build_from_yaml {
 if ($config->{game_folder}) {
     printf "Installing Vulpes to %s\n", $config->{game_folder};
     copy("./Vulpes.dll", $config->{game_folder}."/Vulpes.dll");
-    if ($config->{strings_build}) {
+    # This used to have this cool given - when - default structure.
+    # But turns out that that is expirimental and unstable. I'm sad now.
+    if ($config->{build_type} eq "strings") {
         print "Using strings.dll configuration\n";
-        copy("./VulpesLoader.dll", $config->{game_folder}."/strings.dll");
-    } else {
+        copy("./VulpesLoader.dll",
+            $config->{game_folder}."/strings.dll");
+    } elsif ($config->{build_type} eq "controls") {
         print "Using controls/ configuration\n";
-        copy("./VulpesLoader.dll", $config->{game_folder}."/mods/VulpesLoader.dll");
+        copy("./VulpesLoader.dll",
+            $config->{game_folder}."/controls/VulpesLoader.dll");
+    } elsif ($config->{build_type} eq "mods") {
+        print "Using mods/ configuration\n";
+        copy("./VulpesLoader.dll",
+            $config->{game_folder}."/mods/VulpesLoader.dll");
+    } else {
+        print "Invalid build_type";
```

Comment:
Should print what the invalid build type is, also possibly with a list of valid build types.